### PR TITLE
added skipValidation in upload command

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/45562

## Description
added skipValidation in upload command.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/content/objects/pack_objects/pack.py [hunk 1]
```diff
@@ -328,16 +328,26 @@ def is_server_version_ge(self, client, server_version_to_check):
...
         Upload the pack zip to demisto_client,
         from 6.5 server version we have the option to use skip_verify arg instead of server configuration.
         Args:
             logger (logging.Logger): System logger already initialized.
             client: The demisto_client object of the desired XSOAR machine to upload to.
+            skip_validation: if true will skip upload packs validation.
         Returns:
             The result of the upload command from demisto_client
         """
+        if self.is_server_version_ge(client, '6.6.0'):
```
Comment (guykeller) [inline demisto_sdk/commands/common/content/objects/pack_objects/pack.py:342]: What about greater versions when they will exist?

File: demisto_sdk/commands/upload/README.md [hunk 2]
```diff
@@ -62,6 +62,10 @@ Uploading classifiers to Cortex XSOAR is available from version 6.0.0 and up.
 
     The path to the config file to download all the custom packs from
 
+* **--skip-validation**
+
+    if true will skip upload packs validation, use just when migrate existing custom content to packs.
```
Comment (guykeller) [inline demisto_sdk/commands/upload/README.md:67]: Let's expand this description

Comment (ChanochShayner) [reply demisto_sdk/commands/upload/README.md]: done

File: demisto_sdk/commands/common/content/objects/pack_objects/pack.py [hunk 3]
```diff
@@ -328,16 +328,26 @@ def is_server_version_ge(self, client, server_version_to_check):
...
         from 6.5 server version we have the option to use skip_verify arg instead of server configuration.
         Args:
             logger (logging.Logger): System logger already initialized.
             client: The demisto_client object of the desired XSOAR machine to upload to.
+            skip_validation: if true will skip upload packs validation.
         Returns:
             The result of the upload command from demisto_client
         """
+        if self.is_server_version_ge(client, '6.6.0') and skip_validation:
+            try:
```
Comment (guykeller) [inline demisto_sdk/commands/common/content/objects/pack_objects/pack.py:343]: It seems like for 6.6.0 two uploads will be made, for `is_server_version_ge(client, '6.5.0')` and for 6.6.0.
Is that intentional, or should we change the`if` on line 351 to `elif`?

Comment (ChanochShayner) [reply demisto_sdk/commands/common/content/objects/pack_objects/pack.py:343]: I can replace it to `elif`, but we have the return in line 345

**Review by guykeller — APPROVED** [review]

**Pull request merged**